### ISSUE TITLE
Network information says Russian Federation!?

### ISSUE THREAD

**Issue by PontusTideman** [nodequery/nq-agent#10]
Hi!

On one of my servers the network info tells me the IP is Russian Fed and not the correct one UK.
When checking the IP on maxmind´s site it tells me correct info. So dunno what´s the problem could be?

Best
Pontus

**Comment by PontusTideman** [comment]
No feedback here nor thru e-mail in months...

Would be nice if the developer could check this and give me an answer around this issue. :)

**Comment by sanishan** [comment]
As far as i understand its all based on IP to Location database. if they update ip database on there website, this problem will be gone!

**Comment by PontusTideman** [comment]
![maxmind](https://cloud.githubusercontent.com/assets/9256550/19272257/2a25eb48-8fc8-11e6-928d-c7c44835d010.PNG)
![maxmind1](https://cloud.githubusercontent.com/assets/9256550/19272258/2a323628-8fc8-11e6-8f3b-28aa1708beb4.PNG)

Sorry to say, but as you will see by the images NodeQuery tells me it´s Russian but MaxMinds tells me it´s in London... ^^

**Comment by sanishan** [comment]
Yes because maxmind Database not yet updated as far as i understand, we  have to wait to listen from NodeQuery Itself!

**Comment by bdtech** [comment]
Looks like Italy

http://ipinfo.io/188.164.131.168

5  ae-121-3507.bar2.Milan1.Level3.net (4.69.159.122)  91.595 ms * *
 6  ae-121-3507.bar2.Milan1.Level3.net (4.69.159.122)  91.597 ms 212.73.241.130 (212.73.241.130)  91.073 ms  91.087 ms
 7  217.171.38.134 (217.171.38.134)  93.712 ms  93.673 ms  93.681 ms
 8  mail.nodequery.com (188.164.131.168)  91.627 ms  91.417 ms 217.171.38.134 (217.171.38.134)  93.645 ms
